clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/services/ContainerInformationService.java

```diff
@@ -101,17 +108,30 @@ public List<Map<String, Object>> getHealthStatus(
     // Task-based health
     if (task != null) {
       boolean hasHealthCheck = false;
+      EcsTargetHealth targetHealth = null;
       if (service != null) {
         hasHealthCheck = taskHasHealthCheck(service, accountName, region);
+        LoadBalancer loadBalancer = service.getLoadBalancers().stream().findFirst().orElse(null);
+        String targetGroupKey =
+            Keys.getTargetHealthKey(accountName, region, loadBalancer.getTargetGroupArn());
```

Comment:
loadBalancer can be null, which means getting the target group ARN could through an NPE here.  IF an ECS isn't associated with an LB...